src/components/cart/CartModal/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { Modal } from 'fe-custom-modal';
+import CartItem from '../CartItem';
+import { useContext } from 'react';
+import { CartItemsContext } from '@_context/CartItemsProvider';
+import PaymentDetail from '@_components/common/PaymentDetail';
+import Divider from '@_components/common/Divider';
+import * as S from './style';
+
+interface CartModalProps {
+  isOpened: boolean;
+  modalClose: () => void;
+}
+
+function CartModal({ isOpened, modalClose }: CartModalProps) {
+  const { cartItems } = useContext(CartItemsContext) || { cartItems: [] };
+
+  const totalAmount = cartItems.reduce((acc, cur) => {
+    return acc + cur.product.price * cur.quantity;
+  }, 0);
+
+  return (
+    <>
+      {isOpened && (
+        <Modal
+          modalHeader={{
+            closeButton: { display: false, onClose: modalClose },
+            title: { content: '장바구니', fontSize: '18px', position: 'left' },
+          }}
+          modalFooter={{ confirmButton: { content: '닫기', onConfirm: modalClose } }}
+          modalPosition='bottom'
+          modalSize={{ width: '429px', maxHeight: '90vh' }}
+        >
+          <S.Container>
+            {cartItems &&
+              cartItems.map((item, idx) => {
+                return <CartItem cartItemInfo={item} key={idx} />;
+              })}
+          </S.Container>
+          <S.Result>
+            <Divider />
+            <PaymentDetail title='총 결제 금액' amount={totalAmount} />
+          </S.Result>
+        </Modal>
+      )}
+    </>
+  );
```

Comment:
Fragement와 조건문을 같이 사용하는 대신 아래처럼 적어봐도 좋을거 같네요(취향이라 반영 안 하셔도 됩니다)
```suggestion
 if (!isOpend) return null;

  return (
        <Modal
          modalHeader={{
            closeButton: { display: false, onClose: modalClose },
            title: { content: '장바구니', fontSize: '18px', position: 'left' },
          }}
          modalFooter={{ confirmButton: { content: '닫기', onConfirm: modalClose } }}
          modalPosition='bottom'
          modalSize={{ width: '429px', maxHeight: '90vh' }}
        >
          <S.Container>
            {cartItems &&
              cartItems.map((item, idx) => {
                return <CartItem cartItemInfo={item} key={idx} />;
              })}
          </S.Container>
          <S.Result>
            <Divider />
            <PaymentDetail title='총 결제 금액' amount={totalAmount} />
          </S.Result>
        </Modal>
  );
```

Reply:
생각하지 못한 방법인데 가독성이 더 좋은 것 같습니다😮
적용해보았어요 :)
[98bcf46](https://github.com/woowacourse/react-shopping-products/pull/79/commits/98bcf46cad7d2edb3cecf47d75ba62a5a7ad6ff9)